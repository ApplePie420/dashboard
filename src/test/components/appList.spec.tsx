import { render } from "@testing-library/react";
import AppList, { IAppListProps } from "../../components/appList";

const props: IAppListProps = {
  categories: [
    {
      name: "Category Test",
      items: [
        {
          name: "App Test",
          icon: "bug_report",
          url: "#",
          displayURL: "test",
          newTab: false,
        },
        {
          name: "App Test",
          icon: "bug_report",
          url: "#",
          displayURL: "test",
          newTab: false,
        },
      ],
    },
  ],
  apps: [
    {
      name: "App Test",
      icon: "bug_report",
      url: "#",
      displayURL: "test",
      newTab: false,
    },
  ],
};

it("AppList snapshot test", () => {
  const { asFragment } = render(
    <AppList categories={props.categories} apps={props.apps} />,
  );

  expect(asFragment).toMatchSnapshot();
});
