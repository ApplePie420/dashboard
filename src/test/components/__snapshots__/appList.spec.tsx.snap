// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppList snapshot test 1`] = `[Function]`;
